auth-web/src/store/modules/task.ts

```diff
@@ -36,34 +36,31 @@ export default class TaskModule extends VuexModule {
     // TODO: Add a new API call for fetching counts alone to reduce initial overload - For all calls made in StaffAccountManagement
     @Action({ rawError: true })
     public async syncTasks () {
-      const rejectedAccountsSearchFilter = ConfigHelper.getFromSession(SessionStorageKeys.RejectedAccountsSearchFilter) || ''
-      const pendingAccountsSearchFilter = ConfigHelper.getFromSession(SessionStorageKeys.PendingAccountsSearchFilter) || ''
-
-      if (!pendingAccountsSearchFilter) {
-        let taskFilter: TaskFilterParams = {
-          relationshipStatus: TaskRelationshipStatus.PENDING_STAFF_REVIEW,
-          statuses: [TaskStatus.OPEN, TaskStatus.HOLD]
-        }
-        await this.context.dispatch('fetchTasks', taskFilter)
+      let taskFilter: TaskFilterParams = {
+        relationshipStatus: TaskRelationshipStatus.PENDING_STAFF_REVIEW,
+        statuses: [TaskStatus.OPEN, TaskStatus.HOLD]
       }
+      await this.context.dispatch('fetchTasks', taskFilter)
 
-      if (!rejectedAccountsSearchFilter) {
-        let taskFilter: TaskFilterParams = {
-          relationshipStatus: TaskRelationshipStatus.REJECTED,
-          statuses: [TaskStatus.COMPLETED]
-        }
-        await this.context.dispatch('fetchTasks', taskFilter)
+      taskFilter = {
+        relationshipStatus: TaskRelationshipStatus.REJECTED,
+        statuses: [TaskStatus.COMPLETED]
       }
+      await this.context.dispatch('fetchTasks', taskFilter)
     }
 
     @Action({ rawError: true })
     public async fetchTasks (filterParams: TaskFilterParams) {
       const response = await TaskService.fetchTasks(filterParams)
       if (response?.data) {
         if (filterParams.relationshipStatus === TaskRelationshipStatus.PENDING_STAFF_REVIEW) {
-          this.context.commit('setPendingTasksCount', response.data.total)
+          if (!this.pendingTasksCount) {
```

Comment:
I put these in, because otherwise the totals wont load unless the component is mounted